cli_tools/common/utils/param/param_utils.go

```diff
@@ -169,3 +168,28 @@ func GetRegionalResourcePath(region string, resourceType string, resourceName st
 func GetZonalResourcePath(zone string, resourceType string, resourceName string) string {
 	return getResourcePath(fmt.Sprintf("zones/%v", zone), resourceType, resourceName)
 }
+
+// DisambiguateNetworkAndSubnet returns the URI representation of network and subnet
+// within a given region.
+//
+// There are two goals:
+//
+// 	  a. Explicitly use the 'default' network only when
+//       network is omitted and subnet is empty.
+//    b. Convert bare identifiers to URIs.
+//
+// Rules: https://cloud.google.com/vpc/docs/vpc
+func DisambiguateNetworkAndSubnet(originalNetwork, originalSubnet string, region string) (network string, subnet string) {
```

Comment:
Changed to `ResolveNetworkAndSubnet` (motivated by resolving relative URLs)

About `populate`: I've interpreted populate as modifying the parameters, rather than returning new values. (in the sense of populating empty values).

Overall, though, I'm flexible. Let me know if you'd prefer `populate`!